scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulingMethodImpl.java

```diff
@@ -201,14 +201,21 @@ private int startTasks(Policy currentPolicy, Map<JobId, JobDescriptor> jobMap, M
             //get rmState and update it in scheduling policy
             Set<String> freeResources = getFreeResources(currentPolicy);
             //if there is no free resources, stop it right now without starting any task
```

Comment:
I think that the comment should be moved too.

Reply:
True